packages/common/src/controllers/submissionController.ts

```diff
@@ -19,6 +20,10 @@ const controller = (dependencies: Dependencies) => {
 				const categoryId = Number(req.params.categoryId);
 				const files = req.files as Express.Multer.File[];
 				const organization = req.body.organization;
+
+				// TODO: get userName from auth
```

Comment:
Do we have an issue opened to plan this work?

Reply:
will create a ticket to implement Auth